src/System.ValueTuple/tests/ValueTupleTests.cs

```diff
@@ -1225,6 +1225,107 @@ public static void EightTuplesWithBadRest()
 #endif
         }
 
+#if netcoreapp11
+        [Fact]
+#endif
+        public static void ValueTupleSerialization()
+        {
+            VerifySerialization(new ValueTuple(),
```

Comment:
I thought serialization (XML or binary) was keyed off the same attribute/flag. But it seems the [XmlSerializer ](https://msdn.microsoft.com/en-us/library/ms733901(v=vs.110).aspx) can serialize in other cases too.
Thanks for catching that.
I'll update.